### PR TITLE
Add first thing to title

### PR DESCRIPTION
**Pull request by tgeitz**
Added an item to the newly created menu whatever

### CODE REVIEW

File: src/views/locations/tom/Menu.vue [hunk 1]
```diff
@@ -1,3 +1,5 @@
 <template>
-    <div>Tom's Location Menu</div>
+    <h2>Menu</h2>
+    <h3>"Chikn" Nuggets</h3>
```
Comment (tgeitz) [inline src/views/locations/tom/Menu.vue:3]: "Chikn" is spelled wrong asshole

File: src/views/locations/tom/Menu.vue [hunk 2]
```diff
@@ -1,3 +1,5 @@
...
+    <p>Seitan based chikky nuggies with barbeque sauce</p>
```
Comment (tgeitz) [inline src/views/locations/tom/Menu.vue:4]: This one's ok

File: src/views/locations/tom/Menu.vue (hunk 1 above)
Comment (tgeitz) [reply src/views/locations/tom/Menu.vue:3]: never mind it's ok

**Comment by JohnnyGR33K** [comment]
got my account made

**Pull request merged**